### ISSUE TITLE
Blueprints are broken when the name of a field in the schema changes

### ISSUE THREAD

**Issue by psrpinto** [WordPress/try-wordpress#159]
## Steps to reproduce

- `npm run start:firefox`
- Click "Import Blog Posts"
- Navigate to the screen where one clicks around to fill the blogpost
- Select the title of the post
- In your editor, rename the `title` field in `./schema/subjects/blog-post.json` to something other than `title`. Note that it's `fields.title` we're modifying, not the top-level `title`.
- In your terminal, do `npm run build:schema`
- In your editor, add some blank lines in a TypeScript file and save it, so that the extension reloads
- The following error is displayed

```
Unexpected Application Error!
field is undefined

SingleFieldEditor@moz-extension://1c528d92-66fa-4917-afc2-4888dbede0f5/app.js:375190:1212
```

**Comment by ashfame** [comment]
Blueprints = Subjected Blueprints = SBP? 👍 / 👎

**Comment by psrpinto** [comment]
Would you be up for opening a ticket specifically about what to name Blueprints? I think it would be better to centralize that discussion elsewhere than this ticket. I would like this ticket to be only about fixing this bug.

**Comment by ashfame** [comment]
Sure! Sorry for the drive-by comment. I am deleting the comment and will delete this one as well, so as to remove the noise, as per original intent of this issue.

**Comment by ashfame** [comment]
Documenting what we discussed: This is probably not a consideration if we can just make up a rule of not changing the field once defined. And if ever need to, wiping out the import and starting fresh could be a possible route to move forward.

I personally don't think it warrants data migration as possible handling because of changed fields.

**Comment by psrpinto** [comment]
I believe this one might be fixed, but testing it requires first addressing #174. I'll test this once a fix for #174 lands.

**Comment by psrpinto** [comment]
I can confirm this issue is now fixed.
